src/App.js

```diff
@@ -1,25 +1,36 @@
-import logo from './logo.svg';
-import './App.css';
+//import logo from './logo.svg';
+//import './App.css';
+import {
+    useMemo,
+    useState
+} from 'react';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+export const RandomNumber = ({isShowRandom}) => {
```

Comment:
It seems that this separate component `RandomNumber` should be in a separate file `RandomNumber.jsx`. Maybe, You should transfer it to a separate file? What do You think?

Reply:
I think you are right. I'll do it tomorrow.